Makefile

```diff
@@ -30,6 +30,7 @@ pytest:
 unit:
 	go test -v ./src/rpm
```

Comment:
Are there something like "run all"?

May be simple `go test -v`?

Reply:
Yes, changed to `go test -v ./src/...`.